components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -89,7 +89,7 @@ export class Workspaces extends React.Component<WorkspacesProps, WorkspacesState
                             <div className="w-3/12">Name</div>
                             <div className="w-4/12">Context</div>
                             <div className="w-2/12">Pending Changes</div>
-                            <div className="w-2/12">Last Active</div>
+                            <div className="w-2/12">Last Start</div>
```

Comment:
@gtsiolis I changed this to `Last Start` because that is what the time really shows. `Last Active` was confusing because for running workspaces it seems they should be just not active.